### DISCUSSION TITLE
ARM support

### DISCUSSION THREAD

**Discussion by sosensible** [exoplatform/exo-community#12] [Q&A]
I tried to install the community edition on an ARM based CPU and it doesn't work. Is there an ARM based solution?

**Comment by rdenarie** [comment]
Hello @sosensible 
The community edition can be started with docker according to this documentation : https://exoplatform.github.io/guide/getting-started/start-community.html

As docker is able to run on ARM architecture, I guess that eXo should work.

What is your problem ? Can you give more details ?
Thanks

**Reply by sosensible** [reply]
Containers images are built to run based on a number of key factors.
* Operating System ( Linux, Windows )
* Chip Architecture ( Intel/AMD, ARM, 32bit, 64bit )

Docker runs on ARM, but the containers also have to be "built" to run on that architecture. Not just the eXo containers but things like MySQL.

**Comment by rdenarie** [comment]
Hello @sosensible 
I afraid that we have not ARM based solution to run eXo.
